docs/source/index.rst

```diff
@@ -1,20 +1,20 @@
-.. Compose ML documentation master file, created by
-   sphinx-quickstart on Thu May 30 11:30:19 2019.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+.. toctree::
+    :hidden:
+    :caption: Table of Contents
 
-Welcome to Compose ML's documentation!
-======================================
+    self
+    toc/getting_started
```

Comment:
can we place these different guides in the root for now rather than `toc`?

Reply:
Restructured docs in [this](https://github.com/FeatureLabs/compose-ml/pull/19/commits/9341149a7eaa4c119df4fbd1e2c76e7c5a33e22f#diff-8aa79b83ed86e48962cc94787fc5cb56L5) commit.